data/strings/en.ini

```diff
@@ -1469,6 +1469,11 @@ use_embedded_cs = Use embedded profile
 convert_cs = Convert to working RGB space
 assign_cs = Assign working RGB space
 ask_cs = Ask
+alpha_cs = Alpha && Opacity
+alpha_range_cs = Alpha Range:
+opacity_range_cs = Opacity Range:
+8bit_value_cs = 0-255
+percentage_cs = 0%-100%
```

Comment:
The `_cs` suffix means "color space", which I think it's not the case for these strings.